commands/server/start.go

```diff
@@ -2,44 +2,63 @@ package server
 
 import (
 	"flag"
+	"fmt"
+	"github.com/iov-one/weave/errors"
 
-	"github.com/pkg/errors"
-
+	"github.com/iov-one/weave/coin"
 	"github.com/tendermint/tendermint/abci/server"
 	abci "github.com/tendermint/tendermint/abci/types"
 	cmn "github.com/tendermint/tendermint/libs/common"
 	"github.com/tendermint/tendermint/libs/log"
 )
 
 const (
-	flagBind  = "bind"
-	flagDebug = "debug"
+	flagBind   = "bind"
+	flagDebug  = "debug"
+	flagMinFee = "min_fee"
 )
 
-func parseFlags(args []string) (string, bool, error) {
+type Options struct {
```

Comment:
Please add debug here... maybe we also add home and logger here. No reason for some variables to be options and some to be separate